pkg/resources/resources.go

```diff
@@ -352,11 +360,19 @@ const (
 	MachineDeploymentCRDName = "machinedeployments.cluster.k8s.io"
 	// ClusterCRDName defines the CRD name for cluster objects
 	ClusterCRDName = "clusters.cluster.k8s.io"
+	// GatekeeperConfigCRDName defines the CRD name for gatekeeper config objects
+	GatekeeperConfigCRDName = "configs.config.gatekeeper.sh"
+	// GatekeeperConfigCRDName defines the CRD name for gatekeeper constraint template objects
```

Comment:
This should be `GatekeeperConstraintTemplateCRDName` instead of `GatekeeperConfigCRDName`